build.gradle

```diff
@@ -62,28 +62,28 @@ task uploadArchivesWrapper(dependsOn: createMavenDirectory) << {
     uploadArchives.execute()
 }
 
-// this will upload, but will not first create a directory (which is needed on some servers)
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            repository(url: System.getenv("UPLOAD_MAVEN_REPO_URL")) {
-                authentication(userName: System.getenv("UPLOAD_USERNAME"), password: System.getenv("UPLOAD_PASSWORD"))
-            }
-            pom.version = System.getenv("UPLOAD_VERSION_CBLITE_EKTORP")  // eg, 0.7.2
-            pom.groupId = 'com.couchbase.cblite'
-            pom.artifactId = 'CBLiteEktorp'
-            pom.project {
-                licenses {
-                    license {
-                        name 'Couchbase Community Edition License Agreement'
-                        url 'http://www.couchbase.com/agreement/community'
-                        distribution 'repo'
-                    }
-                }
-            }
-        }
-    }
-}
+//// this will upload, but will not first create a directory (which is needed on some servers)
+//uploadArchives {
+//    repositories {
+//        mavenDeployer {
+//            repository(url: System.getenv("UPLOAD_MAVEN_REPO_URL")) {
+//                authentication(userName: System.getenv("UPLOAD_USERNAME"), password: System.getenv("UPLOAD_PASSWORD"))
+//            }
+//            pom.version = System.getenv("UPLOAD_VERSION_CBLITE_EKTORP")  // eg, 0.7.2
+//            pom.groupId = 'com.couchbase.cblite'
+//            pom.artifactId = 'CBLiteEktorp'
+//            pom.project {
+//                licenses {
+//                    license {
+//                        name 'Couchbase Community Edition License Agreement'
+//                        url 'http://www.couchbase.com/agreement/community'
+//                        distribution 'repo'
+//                    }
+//                }
+//            }
+//        }
+//    }
+//}
```

Comment:
any reason to comment the uploadArchives directive?  does this interfere with anything?